lingui.config.js

```diff
@@ -1,6 +1,6 @@
 /** @type {import('@lingui/conf').LinguiConfig} */
 module.exports = {
-  locales: ['en', 'cs', 'fr', 'hi', 'es'],
+  locales: ['en', 'cs', 'fr', 'hi', 'es', 'jp'],
```

Comment:
'jp' is the country code for Japan.  I'd suggest to use the Japanese language code 'ja' instead.

Reply:
Fixed!